install/amazon_rds/04_configure_the_collector_eks.mdx

```diff
@@ -29,14 +29,16 @@ helm show values pganalyze/pganalyze-collector > values.yaml
 
 Update the serviceAccount section in the values.yaml file to use the role just
 created in the previous step. For the name, use "pganalyze-service-account" as
-specified in the assume role policy:
+specified in the Pod Identity association (for Pod Identity) or the assume role
+policy (for IRSA):
 
 ```yaml
 serviceAccount:
   # -- Specifies whether a service account should be created
   create: true
   # -- Annotations to add to the service account
   annotations:
+    # Add this annotation only if you're using IRSA (not required for Pod Identity)
     eks.amazonaws.com/role-arn: "arn:aws:iam::<aws-account-id>:role/pganalyzeServiceAccountRole"
```

Comment:
If Pod Identity is our preferred mechanism, should we have this commented-out by default? No strong feelings, just thought I'd bring it up.

Reply:
Great point, actually. I think I'll change this to below 👍

```
  # -- Annotations to add to the service account
  annotations: {}
  # Update above annotation if you're using IRSA
  # annotations:
  #   eks.amazonaws.com/role-arn: "arn:aws:iam::<aws-account-id>:role/pganalyzeServiceAccountRole"
```